sections/llorix_one_lite_ribbon_section.php

```diff
@@ -12,15 +12,19 @@
  */
 
 	$ribbon_background = get_theme_mod( 'llorix_one_lite_ribbon_background', apply_filters( 'llorix_one_lite_ribbon_background_filter',llorix_one_lite_get_file( '/images/background-images/parallax-img/parallax-img1.jpg' ) ) );
+    $ribbon_background = apply_filters('llorix_one_lite_translate_single_string', $ribbon_background, 'Ribbon section - background');
 
 	if ( current_user_can( 'edit_theme_options' ) ) {
 	/* translators: %1$s is the customize link %2$s the customize link label */
 	$llorix_one_lite_ribbon_title = get_theme_mod( 'llorix_one_lite_ribbon_title', sprintf( __( 'Change this text in %s','llorix-one-lite' ), sprintf( '<a href="%1$s">%2$s</a>', esc_url( admin_url( 'customize.php?autofocus&#91;control&#93;=llorix_one_lite_ribbon_title' ) ), __( 'Ribbon section','llorix-one-lite' ) ) ) );
+    $llorix_one_lite_ribbon_title = apply_filters('llorix_one_lite_translate_single_string', $llorix_one_lite_ribbon_title, 'Ribbon section - title');
 	} else {
 	$llorix_one_lite_ribbon_title = get_theme_mod( 'llorix_one_lite_ribbon_title' );
+    $llorix_one_lite_ribbon_title = apply_filters('llorix_one_lite_translate_single_string', $llorix_one_lite_ribbon_title, 'Ribbon section - title');
```

Comment:
@stefan-cotitosu Here you have the same statement on both if ... else branches, so it's redundant. It should just be once after the if .. else statement.
This is happening in more files. Please check them out